specs/core/0_beacon-chain.md

```diff
@@ -556,15 +573,15 @@ The following data structures are defined as [SimpleSerialize (SSZ)](https://git
 ```python
 {
     # BLS public key
-    'pubkey': 'bytes48',
+    'pubkey': BLSPubkey,
     # Withdrawal credentials
-    'withdrawal_credentials': 'bytes32',
+    'withdrawal_credentials': Hash,
```

Comment:
@JustinDrake 
Should it be `Bytes32`?

Reply:
Already returned it back to `Bytes32` in this commit https://github.com/ethereum/eth2.0-specs/pull/695/commits/a693737275a4180250b4779e9952f3f84d025609

Reply:
But there are two `withdrawal_credentials`s: one is in `DepositInput` and another one is in `Validator`.

Reply:
Oh, my bad. 
Sure this one should also be `Bytes32`
Thanks for spotting this!

Reply:
I'm tempted to nuke `Bytes32` (which is redundant with `bytes32`) and use `Hash` everywhere. After all, `withdrawal_crendentials` fits with the definition: "result of hashing and its derivatives"

Reply:
I'm fine with it. Commited

Reply:
> After all, `withdrawal_crendentials` fits with the definition: "result of hashing and its derivatives"

Technically the first byte is a version number, but hashes have had built-in version numbers before (see ipfs multi hash) so that's not too terrible...